src/cpp/lpnamer/problem_modifier/AdditionalConstraints.h

```diff
@@ -113,7 +113,7 @@ struct AdditionalConstraints
    * constraints file path
    */
   explicit AdditionalConstraints(
-      std::string const& constraints_file_path,
+      std::string constraints_file_path,
```

Comment:
Why is it not necessary to have a const& ?

Reply:
Because he `std::moves` a copy
1. Pass by copy
2. `std::move` the copy to our data member

A concurrent approach is (usual approch)
1. Pass by const ref
2. Copy

They're equivalent I AFAIK